docs/book/src/SUMMARY.md

```diff
@@ -50,20 +50,22 @@
 
   - [Using a custom type](./component-config-tutorial/custom-type.md)
 
-      - [Adding a new Config Type](./component-config-tutorial/config-type.md)
-      - [Updating main](./component-config-tutorial/updating-main.md)
-      - [Defining your Custom Config](./component-config-tutorial/define-custom-config.md)
```

Comment:
Why are we changing the indention here?
See that it will change the doc and will no longer add the items as a subsection. So, we need to revert these changes